package.json

```diff
@@ -3,8 +3,13 @@
   "version": "0.2.3",
   "description": "UKHSA Spectral ruleset",
   "main": ".spectral.yaml",
+  "type": "module",
```

Comment:
potentially not required if mjs files are being removed.

Reply:
this appears to be necessary as otherwise running tests gives
```
    .../api-guidelines/src/functions/legacy/is-problem-json-schema.js:1505
    export default targetValue => {
    ^^^^^^

    SyntaxError: Unexpected token 'export'
```
@collinsukhsa please feel free to add if I am missing anything